textacy/viz/termite.py

```diff
@@ -138,6 +141,10 @@ def draw_termite_plot(
             raise ValueError(msg)
         highlight_colors = {hc: COLOR_PAIRS[i] for i, hc in enumerate(highlight_cols)}
 
+    if rc_params:
```

Comment:
This block _overwrites_ parts of the global `RC_PARAMS` with whatever is passed in the function call, but it would be safer to modify a local version of the params. Could you do something like

```
_rc_params = RC_PARAMS.copy()
if rc_params:
    _rc_params.update(rc_params)

with plt.rc_context(_rc_params):
    ...
```

instead?

Reply:
@bdewilde yeah sure, it's better indeed :)